bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -1084,7 +1088,7 @@ public ServerConfiguration setMinorCompactionThreshold(double threshold) {
      * @return threshold of major compaction
      */
     public double getMajorCompactionThreshold() {
-        return getDouble(MAJOR_COMPACTION_THRESHOLD, 0.8f);
+        return getDouble(MAJOR_COMPACTION_THRESHOLD, 0.6f);
```

Comment:
why did default change?

Reply:
in this time-based compaction policy, I split the compaction to 3 level, original normal compaction's default value becomes un-distinguish from median and high thresholds.